tests/unit/test_interface.py

```diff
@@ -107,7 +108,8 @@ def test_set_up_dask_client_default() -> None:
     # Shut down a client if it exists
     try:
         client = get_client()
-        client.shutdown()
+        client.shutdown()  # type: ignore [no-untyped-call]
+        time.sleep(30)
```

Comment:
remove this, it's not the source of the bug from before (and we'd never want to sleep 30 seconds in a unit test, anyway)

Reply:
Actually - this will all change when you rebase on the epic branch